src/audits/performance/speed-index-metric.js

```diff
@@ -23,10 +23,10 @@ const TracingProcessor = require('../../lib/traces/tracing-processor');
 
 const FAILURE_MESSAGE = 'Navigation and first paint timings not found.';
 
-// Parameters for log-normal CDF scoring. To see the curve:
-// https://www.desmos.com/calculator/y9qrjhj4e9
-const SCORE_LOCATION = Math.log(5500);
-const SCORE_SHAPE = 0.7;
+// Parameters (in ms) for log-normal CDF scoring. To see the curve:
+// https://www.desmos.com/calculator/mdgjzchijg
+const SCORING_FALLOFF = 1250;
```

Comment:
thanks for adding the description of it to our .getLogNormalDistribution fn

can we use the full name for this variable? SCORING_POINT_OF_DIMINISHING_RETURNS
it's unwieldy but I'm into keeping this part very clear and reducing our vocabulary where possible